zproject/settings.py

```diff
@@ -1075,3 +1075,4 @@ def get_secret(key):
 PROFILE_ALL_REQUESTS = False
 
 CROSS_REALM_BOT_EMAILS = set(('feedback@zulip.com', 'notification-bot@zulip.com'))
+ARCHIVED_DATA_RETENTION_DAYS = 30
```

Comment:
This needs to be user-configurable and thus should be added to DEFAULT_SETTINGS rather than set unconditionally here; see http://zulip.readthedocs.io/en/latest/settings.html for details on why.